_posts/2022-04-26-fivetran-custom-connector.md

```diff
@@ -0,0 +1,338 @@
+---
+layout: post
+title: FiveTran Paginated Custom Connector
+date: 2022-05-26
+tags: fivetran custom connector custom-connector pagination getmore state-management
+author: Joseph Miocevich
+image: img/fivetran_logo.png
+description:
+  Overview of a custom fivetran connector with pagination
+---
+
+![FiveTran Paginated Custom Connectors](/img/fivetran_logo.png)
+
+<br/>
+
+Understanding and debugging your FiveTran custom connector is not straightfoward, with many online tutorials excluding the key component of data API's... pagination. After reading this blog you will have a better understanding of the expected behaviour between FiveTran and your custom connector when pagination is thrown into the mix.
+
+## Overview
+
+A FiveTran custom connector is a cloud hosted function that can be set up on a cloud platform of your choice, which allows you to load data into FiveTran with a API FiveTran does not natively support. 
+
+Like standard connectors, custom connectors have a number of benefits:
+>
```

Comment:
Does this format the quote properly?  Thought it needed to be on each line.

Reply:
Formats properly :)